docs/getting-started/integration.md

```diff
@@ -79,22 +93,43 @@ Chart.register(
   Tooltip,
   SubTitle
 );
-
-var myChart = new Chart(ctx, {...});
 ```
 
-A short registration format is also available to quickly register everything.
+### How to choose the imports you require.
+
+The options are categorised (inspect `registerables`) into controllers, elements, plugins, scales. You can pick and choose many of these, e.g. if you are not going to use tooltips, don't import and register the `Tooltip` plugin. But each type of chart has its own bare-minimum requirements (typically the type's controller and a suitable element):
+
+* Bar chart
+   * `BarController`
+   * `BarElement`
+   * `CategoryScale`
+   * One of `LinearScale` (default) or `LogarithmicScale`
+* Pie chart
+   * `PieController`
+   * `ArcElement`
+* Doughnut chart
+   * `DoughnutController`
+   * `ArcElement`
+* ...@todo...
+
+The plugin imports are
+
+* [`Decimation`](../configuration/decimation.md)
+* `Filler` - used to fill the area under the line, see [Area charts](../charts/area.md)
+* `Legend`
+* `Title`
+* `Tooltip`
+* `SubTitle`
+
+The scale imports are:
+
+* `CategoryScale`
```

Comment:
A number of charts also use the `CategoryScale` as the default X axis

Reply:
To add on this I dont think there really is a default scale, since bar and line use `category` for x, scatter uses `linear`, doughnut and pie dont use any and radar and polar use a `radiallinear` 

So for the basic implementation of all the charts you would need either zero or half of the scales already so there is not really a single default one

Reply:
I think it would be better to list "default scales" for each chart type in the "bare-minimum" list above.